go/ngql/types.go

```diff
@@ -203,10 +203,17 @@ func structToGQLObject(nomsType *types.Type, tm *typeMap) *graphql.Object {
 			structDesc.IterFields(func(name string, nomsFieldType *types.Type) {
 				fieldType := nomsTypeToGraphQLType(nomsFieldType, false, tm)
 
+				if nonNull, ok := fieldType.(*graphql.NonNull); ok {
```

Comment:
Did you want all types to be nullable? It is only struct fields that need this hack.